cddl_gen/cddl_gen.py

```diff
@@ -220,10 +220,15 @@ def strip_comments(instr):
         comment_regex = r"\;.*?\n"
```

Comment:
If you `re.compile` this string into a class variable, you can do `comment_regex.sub('', instr)` to get faster run times with the function and all others using `comment_regex`. Highly recommend pre-compiling all regex strings for getting that bonus when possible.

Reply:
Thanks. The two regexes you commented on are only used once, so precompiling is not a gain. It would be huge gain in get_value(), though, so I'll do that at some point.